SpatialGDK/Source/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1047,7 +1051,7 @@ void USpatialNetDriver::TickFlush(float DeltaTime)
 #if USE_SERVER_PERF_COUNTERS
 	double ServerReplicateActorsTimeMs = 0.0f;
 #endif // USE_SERVER_PERF_COUNTERS
-	if (IsServer() && ClientConnections.Num() > 0)
+	if (IsServer() && ClientConnections.Num() > 0 && Connection->IsConnected())
```

Comment:
Drive-by?

Reply:
Yes. I'm not sure how this ever worked without that condition.